client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -15,6 +15,8 @@ import { getStreamUrl } from 'reader/route';
 import EmailSettings from './email-settings';
 import { getSiteName, getSiteUrl } from 'reader/get-helpers';
 
+const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( /\/$/, '' );
```

Comment:
sure, i'll add a comment.  

> & is it worth moving out to somewhere for reuse?

Not sure where it should live just yet.  I think this is the only place that needs it for now.